build/jenkins_build.bat

```diff
@@ -105,9 +105,17 @@ if %errorlevel% neq 0 (
 
 REM Copy the install script across
 cd %BASEDIR%
-copy /Y %BASEDIR%\install_client.bat %INSTALLDIR%
+copy /Y %BASEDIR%\install_gui_and_python.bat %INSTALLDIR%
 if %errorlevel% neq 0 (
```

Comment:
If we don't copy `install_client.bat` then there needs to be a PR to edit the IBEX install script as i expect that would still be calling `install_client.bat`